code/config/lib/Loot/Config/Record.hs

```diff
@@ -121,6 +180,14 @@ type family ValuesConstrained c is :: Constraint where
         ( ValuesConstrained c us
         , ValuesConstrained c is
         )
+    ValuesConstrained c ((l ::+ us) ': is) =
+        ( ValuesConstrained c (SumSelection l : ToBranches us)
```

Comment:
I agree with you, I remember adding that constraint because something was not compiling and I thought that it made sense: `SumSelection l` is effectively another `option` so that should be checked as well, right?

Apparently not, I removed it now and tested it a bit, it all seems to compile and work as expected.